src/sage/combinat/root_system/root_lattice_realizations.py

```diff
@@ -2867,32 +2866,34 @@ def plot_alcove_walk(self, word, start=None, foldings=None, color="orange", **op
 
                 sage: L = RootSystem(["A",2,1]).ambient_space()
                 sage: w1 = [0,2,1,2,0,2,1,0,2,1,2,1,2,0,2,0,1,2,0]
-                sage: p = L.plot_alcoves(bounding_box=5)           # long time (5s)
-                sage: p += L.plot_alcove_walk(w1)                  # long time
-                sage: p                                            # long time
+                sage: p = L.plot_alcoves(bounding_box=5)           # long time (5s)     # optional - sage.plot sage.symbolic
+                sage: p += L.plot_alcove_walk(w1)                  # long time          # optional - sage.plot sage.symbolic
+                sage: p                                            # long time          # optional - sage.plot sage.symbolic
                 Graphics object consisting of 375 graphics primitives
 
             The same plot with another alcove walk::
 
                 sage: w2 = [2,1,2,0,2,0,2,1,2,0,1,2,1,2,1,0,1,2,0,2,0,1,2,0,2]
-                sage: p += L.plot_alcove_walk(w2, color="orange")  # long time
+                sage: p += L.plot_alcove_walk(w2, color="orange")  # long time          # optional - sage.plot sage.symbolic
```

Comment:
considering the number of times we have `sage.plot` and `sage.symbolic`, both could be at the top of the file.

Reply:
Just went through this file systematically (and made some cosmetic changes in d15a157).

This is fundamental code for root systems; I wouldn't like testing of it to depend on plotting (and symbolics) packages.

One thing that could easily be done though is to move all plotting methods to a separate file and lazy-import from there. Then that file could get a file-level `# optional`.